dash/Dashboard.py

```diff
@@ -255,11 +256,13 @@ def plot_pedestrian_cartesian_chart(self, center_id, vehicles, pedestrians, refe
         fig = plt.figure(Dashboard.CART_FIG_ID)
         plt.cla()
 
+        c_plot_area = self.sim_traffic.origin[3] if len(self.sim_traffic.origin) > 3 else CPLOT_SIZE
```

Comment:
is this check ever false `len(self.sim_traffic.origin) > 3`? Origin always have 4 members...